vendor.conf

```diff
@@ -23,7 +23,7 @@ github.com/RackSec/srslog 456df3a81436d29ba874f3590eeeee25d666f8a5
 github.com/imdario/mergo 0.2.1
 
 #get libnetwork packages
-github.com/docker/libnetwork 1a019214c9cb80bd56219e5d6994a22caf302895
+github.com/docker/libnetwork bb4fe0b6f806412fc671348bfdc42a693c00a0ae
```

Comment:
We need it in 17.04.

But I was hoping to bring it in along with https://github.com/docker/libnetwork/pull/1678 if that one and https://github.com/docker/swarmkit/pull/2028 get merged in time.

Maybe better be safe and carry it in now along with this vendoring.